webapp/test_results_handler.go

```diff
@@ -90,36 +82,46 @@ func parseTestResultsUIFilter(r *http.Request) (filter testResultsUIFilter, err
 	}
 	ctx := shared.NewAppEngineContext(r)
 
-	if testRunFilter.IsDefaultQuery() {
-		experimentalByDefault := shared.IsFeatureEnabled(ctx, "experimentalByDefault")
-		experimentalAlignedExceptEdge := shared.IsFeatureEnabled(ctx, "experimentalAlignedExceptEdge")
-		masterRunsOnly := shared.IsFeatureEnabled(ctx, "masterRunsOnly")
-		if experimentalByDefault {
-			if experimentalAlignedExceptEdge {
-				testRunFilter = testRunFilter.OrAlignedExperimentalRunsExceptEdge()
+	filter.PR, err = shared.ParsePRParam(r)
+	if err != nil {
+		return filter, err
+	} else if filter.PR != nil {
+		one := 1
```

Comment:
Haha, one of the least debatable lines of code I've seen in a while :D

Reply:
I find it extremely frustrating that you can't point to the address of a constant.